src/Application.vala

```diff
@@ -32,6 +32,10 @@ public class InitialSetup.Application : Gtk.Application {
 
         var main_window = new MainWindow (this);
         main_window.show_all ();
+
+        var css_provider = new Gtk.CssProvider ();
+        css_provider.load_from_resource ("io/elementary/initial-setup/application.css");
+        Gtk.StyleContext.add_provider_for_screen (Gdk.Screen.get_default (), css_provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
```

Comment:
Can we move this into the window class itself and use `window.get_screen ()` here?